front/src/containers/pages/DataListView.js

```diff
@@ -66,7 +69,8 @@ class DataListView extends React.Component {
                   />
                   <div
                     className="glyph-icon simple-icon-trash delete-action-icon"
-                    onClick={onDeleteItem}
+                    // onClick={onDeleteItem}
```

Comment:
Could you delete this comment pls?

Reply:
habla en español papa

Reply:
JAJAJA me olvido que no estoy en el laburo